File: 268-MissingNumber/268-MissingNumber.py
# Last updated: 6/25/2026, 9:14:46 AM
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        res=len(nums)

        for i in range (len(nums)):
            res^=i
            res^=nums[i]

        return res

File: 268-MissingNumber/268-MissingNumber.cpp
// Last updated: 6/25/2026, 9:14:49 AM
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int number = 0;
        for(int i=0; i<nums.size(); i++) number += i-nums[i];
        return number+nums.size();
    }
};